src/frontend/js/components/SaleTunnel/components/SaleTunnelStepResume/index.tsx

```diff
@@ -18,34 +21,66 @@ const messages = defineMessages({
     description: "Text to remind that order's invoice will be send by email soon",
     id: 'components.SaleTunnelStepResume.successDetailMessage',
   },
+  successDetailSignatureMessage: {
+    defaultMessage: 'In order to enroll to course runs you first need to sign the contract.',
```

Comment:
Let's use training contract instead of contract
```
    defaultMessage: 'In order to enroll to course runs you first need to sign the training contract.',
```